client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -105,6 +126,28 @@ export function SitesDashboard( { queryParams: { search, status = 'all' } }: Sit
 							{ displayMode === 'tile' && (
 								<SitesGrid isLoading={ isLoading } sites={ filteredSites } />
 							) }
+							{ selectedStatus.hiddenCount && (
+								<HiddenSitesMessage>
+									{ translate(
+										'%(hiddenSitesCount)d more hidden site. {{a}}Change{{/a}}.{{br/}}Use search to access it.',
+										'%(hiddenSitesCount)d more hidden sites. {{a}}Change{{/a}}.{{br/}}Use search to access them.',
```

Comment:
These strings are reused from `<SiteSelector>` but I don't think they're quite right...

Reply:
I think it makes sense to rename "Change" CTA as here it performs different action than in the sidebar selector, where it points to a page that allows changing visibility settings.

Maybe "Show all" as this is already used in other places and translated?

Reply:
I've changed the CTA to "Show all"